src/main/java/teamroots/embers/tileentity/TileEntityDawnstoneAnvil.java

```diff
@@ -35,124 +36,122 @@
 import java.util.Random;
 
 public class TileEntityDawnstoneAnvil extends TileEntity implements ITileEntityBase, IHammerable, ISpecialFilter {
-	public static int MAX_HITS = 40;
-	int ticksExisted = 0;
-	int progress = 0;
-	public ItemStackHandler inventory = new ItemStackHandler(2){
+    public static int MAX_HITS = ConfigMachine.DAWN_STONE_ANVIL_CATEGORY.maxHits;
+    int ticksExisted = 0;
+    int progress = 0;
+    public ItemStackHandler inventory = new ItemStackHandler(2) {
         @Override
         protected void onContentsChanged(int slot) {
-        	TileEntityDawnstoneAnvil.this.markDirty();
+            TileEntityDawnstoneAnvil.this.markDirty();
         }
 
-		@Override
-		protected int getStackLimit(int slot, @Nonnull ItemStack stack) {
-			return 1; //CURSED
-		}
-	};
-	Random random = new Random();
-	
-	public TileEntityDawnstoneAnvil(){
-		super();
-	}
-	
-	@Override
-	public NBTTagCompound writeToNBT(NBTTagCompound tag){
-		super.writeToNBT(tag);
-		tag.setTag("inventory", inventory.serializeNBT());
-		tag.setInteger("progress", progress);
-		return tag;
-	}
-	
-	@Override
-	public void readFromNBT(NBTTagCompound tag){
-		super.readFromNBT(tag);
-		inventory.deserializeNBT(tag.getCompoundTag("inventory"));
-		progress = tag.getInteger("progress");
-	}
+        @Override
+        protected int getStackLimit(int slot, @Nonnull ItemStack stack) {
+            return 1; //CURSED
+        }
+    };
+    Random random = new Random();
 
-	@Override
-	public NBTTagCompound getUpdateTag() {
-		return writeToNBT(new NBTTagCompound());
-	}
+    public TileEntityDawnstoneAnvil() {
```

Comment:
Can be removed

Reply:
Could you explain to me why this can be removed without causing any issues?

Reply:
If there is no constructor, the super constructor is called automatically.